src/macros-by-example.md

```diff
@@ -195,6 +195,220 @@ compiler knows how to expand them properly:
     not have the same number. This requirement applies to every layer of nested
     repetitions.
 
+## Meta variable expressions
+
+Give access to additional metadata about meta variables that otherwise would be difficult or even impossible to get through elements such as `count` or `index`.
```

Comment:
> elements such as `count` or `index`.

The wording implies that `count` and `index` are "elements" through which it would be difficult or impossible to get metadata, but presumably that's not the intention.